### PR TITLE
[docs] add unknown route to troubleshooting

### PR DESCRIPTION
**Pull request by bmorelli25**
For https://github.com/elastic/kibana/issues/51733.

This is a rough copy/pasta of @SylvainJuge's [comment on discuss](https://discuss.elastic.co/t/how-solve-get-unknown-route-from-apm-java/209364). I've made a few changes to adapt it for documentation.

### CODE REVIEW

File: docs/troubleshooting.asciidoc [hunk 1]
```diff
@@ -149,6 +149,42 @@ If the APM server does not respond successfully,
...
+It's possible to change the default transaction naming to use the URL path instead.
+See <<config-use-path-as-transaction-name>> for more information.
++
+Unfortunately, this may create a lot of duplicate transactions if they have similar paths.
+For example, in `/usr/{id}`, where `{id}` is the user ID,
+you can end up with as many transactions as there are users.
+You can mitigate this by using <<config-url-groups>>,
+which will allow the use of wildcards in transaction URLs.
++
+The final option is to set transaction names using our API:
```
Comment (eyalkoren) [inline docs/troubleshooting.asciidoc]: In the HTML, this section is nested within the `Requests do not reach a servlet`. Doesn't seem right.
Additionally, I would suggest something like:
```suggestion
If the proposed fixes do not solve the problem, or if a custom name is required, transaction names can be set manually throughout the request handling flow using our API:
```

File: docs/troubleshooting.asciidoc [hunk 2]
```diff
@@ -149,6 +149,42 @@ If the APM server does not respond successfully,
...
+
+It's possible to change the default transaction naming to use the URL path instead.
+See <<config-use-path-as-transaction-name>> for more information.
++
+Unfortunately, this may create a lot of duplicate transactions if they have similar paths.
+For example, in `/usr/{id}`, where `{id}` is the user ID,
+you can end up with as many transactions as there are users.
+You can mitigate this by using <<config-url-groups>>,
+which will allow the use of wildcards in transaction URLs.
++
```
Comment (bmorelli25) [inline docs/troubleshooting.asciidoc]:
```suggestion

```

File: docs/troubleshooting.asciidoc [hunk 3]
```diff
@@ -149,6 +149,41 @@ If the APM server does not respond successfully,
...
+Requests do not reach a servlet::
+
+It's possible to change the default transaction naming to use the URL path instead.
+See <<config-use-path-as-transaction-name>> for more information.
++
+Unfortunately, this may create a lot of duplicate transactions if they have similar paths.
+For example, in `/usr/{id}`, where `{id}` is the user ID,
+you can end up with as many transactions as there are users.
+You can mitigate this by using <<config-url-groups>>,
+which will allow the use of wildcards in transaction URLs.
```
Comment (bmorelli25) [inline docs/troubleshooting.asciidoc]:
```suggestion
which will allow the use of wildcards in transaction URLs.

```

**Comment by bmorelli25** [comment]
Thanks @eyalkoren! Updated with your suggested text change and removed the `+` to remove the indentation.

**Pull request merged**